redis/src/types.rs

```diff
@@ -998,7 +1006,10 @@ impl RedisError {
         Self { repr }
     }
 
-    pub(crate) fn retry_method(&self) -> RetryMethod {
+    /// Specifies what method (if any) should be used to retry this request.
```

Comment:
docs should clarify that this isn't precise, but just rough guidelines / recommendations, and that the library uses more considerations internally than just the error kind.